packages/snyk/package.yaml

```
@@ -0,0 +1,36 @@
+---
+name: snyk
+description: |
+  Snyk CLI scans and monitors your projects for security vulnerabilities
+homepage: https://github.com/snyk/cli
+licenses:
+  - Apache-2.0
+languages:
+  - Docker
+  - Go
+  - Helm
+  - JavaScript
+  - Python
+  - Ruby
+  - Rust
+  - Terraform
+  - TypeScript
+categories:
+  - Linter
+
+source:
+  id: pkg:github/snyk/cli@v1.1243.0
+  asset:
+    - target: darwin_x64
+      file: snyk-macos
+    - target: darwin_arm64
+      file: snyk-macos-arm64
+    - target: linux_x64
+      file: snyk-linux
+    - target: linux_arm64
```

I believe these are built for GNU systems:
```suggestion
    - target: linux_x64_gnu
      file: snyk-linux
    - target: linux_arm64_gnu
```